### PR TITLE
Add bracci2019 benchmark (competition)

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds the Bracci2019 Benchmark for the 2024 competition, as created by: [costantinoai](https://github.com/costantinoai).

This consists of a single new benchmark, `Bracci2019.anteriorVTC-rdm`

### CODE REVIEW

**Review by costantinoai — APPROVED** [review]

File: brainscore_vision/benchmarks/bracci2019/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+            "anteriorVTC": (2, "IT"),
+        }
+
+        assert (
+            self._region in self._roi_map
+        ), "The ROI to compare must either ['V1', 'posteriorVTC', or 'anteriorVTC']"
+
+        # Call the parent class constructor to complete initialization
+        super(_Bracci2019RSA, self).__init__(
+            identifier=f"Bracci2019_RSA-{region}",
```
Comment (mschrimpf) [inline brainscore_vision/benchmarks/bracci2019/benchmark.py:94]: mini convention request for `{dataset}-{metric}`
```suggestion
            identifier=f"Bracci2019.{region}-rdm",
```

File: brainscore_vision/data/bracci2019/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+natsort
+logging
+scipy
+numpy
+pathlib
```
Comment (mschrimpf) [inline brainscore_vision/data/bracci2019/requirements.txt:5]: are these all needed for data *use* or only for data *packaging*? If it's only the later, no need to include the requirements here

**Review by mschrimpf — APPROVED** [review]

File: brainscore_vision/benchmarks/bracci2019/benchmark.py (hunk 1 above)
Comment (costantinoai) [reply brainscore_vision/benchmarks/bracci2019/benchmark.py:94]: @mike-ferguson Feel free to make these suggested changes before merging -- sorry, I do not have write access. I am not sure, but probably a similar change is needed in `brainscore_vision/benchmarks/bracci2019/__init_.py`?

**Comment by mike-ferguson** [comment]
@mschrimpf @costantinoai This benchmark now produces scores and can be merged when tests pass!

**Pull request merged**